source/_posts/2019/2019-11-18-how-to-delegate-code-reviews-to-ci.md

```diff
@@ -71,20 +71,47 @@ vendor/bin/rector process src --config rector-ci.yaml --dry-run
 
 I've prepared a demo with PHP code and a testing pipeline for all widely used CI services.
 
-- [Github + Travis CI](#1-github-travis-ci)
+- [GitHub Actions](#0-github-actions)
+- [GitHub + Travis CI](#1-github-travis-ci)
 - [Gitlab CI](#2-gitlab-ci)
 - [BitBucket](#3-bitbucket)
 
 <br>
 
-- ~~Github Actions~~
-*Do you know how to work with Github Actions? Please let me know how would script look like, so we can complete the list.*
+There you'll find all the configuration you need to **let your CI do code-reviews**.
+
+### 0. GitHub Actions
 
 <br>
 
-There you'll find all the configuration you need to **let your CI do code-reviews**.
+<a href="http://github.com/tomasvotruba/rector-ci-demo" class="btn btn-info">Repository</a>
+<a href="https://github.com/TomasVotruba/rector-ci-demo/actions/" class="btn btn-success ml-3">CI Feedback</a>
+
+```yaml
+# .github/workflows/php.yml
+name: Rector Code Review
+
+on: [push]
+
+jobs:
+
+  build:
```

Comment:
Please use 4-space indent, as in other configs. Just for consistence.

Reply:
Oh, right, sorry I did not notice, thanks for pointing out.

Reply:
No troubles